main.py

```diff
@@ -0,0 +1,5 @@
+"""Google cloud function entry points."""
```

Comment:
Does this need to exist at the top level of the repo? Why can't we have this under `probe_scraper`? The docs here should contain some commentary about that.

Reply:
it does, i will expand the docstring to explain